doc/delaytrain.txt

```diff
@@ -45,6 +45,8 @@ following configuration (default settings included):
             ['nv'] = {'h', 'j', 'k', 'l'},
             ['nvi'] = {'<Left>', '<Down>', '<Up>', '<Right>'},
         },
+        ignore_filestypes = {}, -- Example: set to {"help", "NvimTree"} to
+                               -- disable the plugin for help and NvimTree
```

Comment:
Quick fixes here:

1. Typo in `ignore_filestypes`
2. Would you mind aligning the comment delimiter in line 49 to match 48? Unless they align properly in the help doc.
3. Could you add this to the README and add a quick blurb in both places on how to find a buffer's filetype with `:set ft`?